weekly-q-and-a-sessions/16-12-2021.md

```diff
@@ -0,0 +1,31 @@
+
+# Thursday October 21, 2021
+
+## Attendees
+
+- Adi Dick
+- Benjamin Akar
+- Richard Walker
+- Martine Trulsrud
+
+## Notes
+
+### Bug Reports: Martine Trulsrud
+
+Martine reported a couple bugs in the React components after working with Martin the day before
+
+#### Text field component
+* If you use affix and have too many characters in the main field, the affix gets pushed down
+* prefix is read last by screen reader
```

Comment:
The same goes for suffix. The question is whether we want screenreaders to read the affix before targetting the input.